ant-cli/src/commands.rs

```diff
@@ -70,6 +70,9 @@ pub enum FileCmd {
         /// Upload the file as public. Everyone can see public data on the Network.
         #[arg(short, long)]
         public: bool,
+        /// Create and upload archive after file upload. This enables traditional archive functionality.
```

Comment:
Could we perhaps state the purpose of the archive here? I think it's for storing metadata.

We could also mention here that it will cost more for the archive.